### PR TITLE
Feature/5706 add associated records

### PR DESCRIPTION
**Pull request by HavierD**

### CODE REVIEW

File: src/components/list/listItem/CollapseItem.tsx [hunk 1]
```diff
@@ -102,4 +102,4 @@ const CollapseFrame: React.FC<CollapseFrameProps> = ({
   );
 };
 
-export default CollapseFrame;
+export default CollapseItem;
```
Comment (vietnguyengit) [inline src/components/list/listItem/CollapseItem.tsx:105]: you have "PlainCollapseList" in the other file, why this isn't "PlainCollapseItem" but just "CollapseItem" ? What happens if you have "CollapseItem" and "CollapseList", the word "plain" is really confusing to me.

File: src/components/list/AssociatedRecordList.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        </CollapseItem>
+      );
+    });
+    return components;
+  }, [openRecord, records]);
+
+  return <ExpandableList title={title} childrenList={collapseComponents} />;
+};
+
+export default AssociatedRecordList;
```
Comment (vietnguyengit) [inline src/components/list/AssociatedRecordList.tsx:44]: LGTM

**Review by vietnguyengit — CHANGES_REQUESTED** [review]
code are good, but just naming stuff to address as usual @HavierD

File: src/components/list/listItem/PlainTextItem.tsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import { Grid } from "@mui/material";
 import React from "react";
-import { margin } from "../../../../styles/constants";
+import { margin } from "../../../styles/constants";
 
-interface PlainTextFragmentProps {
+interface PlainTextItemProps {
```
[the remark this answers is not in the document]
Comment (HavierD) [reply src/components/list/listItem/PlainTextItem.tsx:5]: Thank you. At the beginning, i decided to name these components on detail page like: PlainText, MediaText, etc.. but then i changed the implementation so the "Plain" prefix looks strange. I will remove it.

[the remark this answers is not in the document]
Comment (vietnguyengit) [reply src/components/list/listItem/PlainTextItem.tsx:5]: cool, they are all just "text" so it's good that you don't have a bunch of different "text" components.

**Review by vietnguyengit — APPROVED** [review]

**Pull request merged**